app/src/main/java/ch/epfl/sdp/blindly/profile_setup/ProfileShowMe.kt

```diff
@@ -26,6 +40,15 @@ class ProfileShowMe : AppCompatActivity() {
             findViewById<TextView>(R.id.warning_p6).visibility = View.VISIBLE
         } else {
             val intent = Intent(this, ProfilePassions::class.java)
+            val showMe = findViewById<RadioButton>(radioGroup.checkedRadioButtonId)
+                    .text.toString().toLowerCase().capitalize()
```

Comment:
Cirrus warning about the fact that using the default locale might cause bugs. If I understood it well, I think it is better to write `capitalize(myString)` instead of `myString.capitalize()`.